app/models/user.rb

```diff
@@ -47,6 +47,27 @@ class User
     validates :username, presence: true
   end
 
+  def self.valid_google_domain?(email)
+    return true if Errbit::Config.google_authorized_domains.nil?
+    match_data = /.+@(?<domain>.+)/.match(email)
+    return false if match_data.nil?
+    Errbit::Config.google_authorized_domains.split(",").include?(match_data[:domain]) ? true : false
```

Comment:
There's no need for the ternary because the return value of `include?` here is already correct.